dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/util/json/JsonPrinter.java

```diff
@@ -854,4 +855,21 @@ public static JsonObjectBuilder mapToObject(Map<String, String> in) {
         in.keySet().forEach(k -> b.add(k, in.get(k)));
         return b;
     }
+
+
+    private static String getLicenseName(FileMetadata fmd) {
+        Optional<License> license = Optional.of(fmd.getTermsOfUse().getLicense());
+        if(license.isPresent()) {
```

Comment:
There is no point in using monad if you are exiting it right away. Use mapping and then orElse(StringUtils.EMPTY);